com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/cubrid/trans/ToCUBRIDDataConverterFacade.java

```diff
@@ -140,7 +140,7 @@ public Object convert(Object obj, DataTypeInstance dti,
 		Object dataTypeID = dataTypeHelper.getCUBRIDDataTypeID(dti.getName());
 		final AbstractDataConverter cvter = converterMap.get(dataTypeID);
 		if (cvter != null) {
-			Object value = obj;
+			Object value = dataTypeHelper.getCUBRIDDataSetByDataTypeID (obj, dti.getName()) ;
```

Comment:
* In this line, Every column's dataType is checked whether it is NUMERIC type or not. It can causes a performance degradation.
* Please, refer to "NumericConverter".  NumericConverter class handles for NUMERIC DataType.